README.md

```diff
@@ -1,7 +1,22 @@
-# Carapace
+# ![Carapace](https://user-images.githubusercontent.com/2371345/65792037-4f388380-e139-11e9-83ae-501282063c6e.png)
+[![Build Status](https://travis-ci.com/Islandora-CLAW/Carapace.png?branch=8.x-1.x)](https://travis-ci.com/Islandora-CLAW/Carapace)
+[![Contribution Guidelines](http://img.shields.io/badge/CONTRIBUTING-Guidelines-blue.svg)](./CONTRIBUTING.md)
+[![LICENSE](https://img.shields.io/badge/license-GPLv2-blue.svg?style=flat-square)](./LICENSE)
+
+## Introduction
+
 An AdaptiveTheme sub-theme customized to optimize the Islandora experience.
 
-INSTALLATION
+## Requirements
+
+This module requires the following modules/libraries:
+
+* [AdaptiveTheme framework](https://www.drupal.org/project/adaptivetheme) 
+* [AdaptiveTheme tools](https://www.drupal.org/project/at_tools) 
```

Comment:
As this PR got hung up during the paged content merge, now it requires some more changes. But I want @dannylamb to just confirm my statements.

Statement 1: We no longer require `at_tools`

Reply:
Statement 1.1: The below lines 23-26 can be removed as they refer to `at_tools`